### PR TITLE
Libcperciva import

### PR DESCRIPTION
**Pull request by gperciva**

### CODE REVIEW

File: libcperciva/cpusupport/Build/cpusupport-HWCAP-GETAUXVAL.c [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#include <sys/auxv.h>
```
Comment (cperciva) [inline libcperciva/cpusupport/Build/cpusupport-HWCAP-GETAUXVAL.c:1]: Is there a reason to have this .c file, given that tarsnap doesn't use it?

Comment (gperciva) [reply libcperciva/cpusupport/Build/cpusupport-HWCAP-GETAUXVAL.c:1]: Oops, sorry.  I had a mental note to remove it from the non-kivaloo projects.

Comment (gperciva) [reply libcperciva/cpusupport/Build/cpusupport-HWCAP-GETAUXVAL.c:1]: ... well, actually my mental note was about not including cpusupport-ARM-CRC32.c, and I was successful at that.  I just neglected to apply that to HWCAP-GETAUXVAL.c as well.

**Comment by gperciva** [comment]
Updated to not include cpusupport-HWCAP-GETAUXVAL.c.

**Pull request merged**